infrastructure/scripts/setup-search.sh

```diff
@@ -35,18 +35,32 @@ OUTPUT=$(terraform output -json)
 popd
 
 {
-    echo "API_ADMIN_KEY=$(echo "$OUTPUT" | jq .search_admin_key.value --raw-output)"
+    echo "API_ADMIN_KEY=\"$(echo "$OUTPUT" | jq .search_admin_key.value --raw-output)\""
```

Comment:
Might still be worth having the `update-search-env` as a separate script so that you can populate your local env even when the search service components are already configured?